src/aind_nwb_utils/utils.py

```diff
@@ -213,13 +193,43 @@ def get_subject_nwb_object(data_path: Path) -> Subject:
         except Exception:
             pass
 
+    if session_start_date_time.tzinfo is None:
+        pacific = pytz.timezone("US/Pacific")
```

Comment:
maybe `default_zone="US/Pacific")` could be exposed as an input to this function